roadmap.md

```diff
@@ -22,21 +22,26 @@ We must finish the Taskcluster migration. Supporting both Buildbot and
 Taskcluster within automation is a substantial drag on agility and
 productivity, and is costly in its resource consumption.
 
-### Sunsetting Deprecated Features
+The remaining bits of the migration include:
+ * Porting Windows tests to run natively in Taskcluster
+ * Porting Windows and Linux tasks that must run on hardware (Talos, etc.) to run natively in Taskcluster
```

Comment:
Isn't it more about standing up hardware workers, rather than generating task definitions? I think we're probably already able to generate the task definitions (which I see as the porting part) whereas we're not in a position to manage/maintain hardware worker pools yet. Correct me if I'm wrong about that.